src/parameterized_tendencies/radiation/radiation.jl

```diff
@@ -256,11 +272,21 @@ function rrtmgp_model_kwargs(
         end
 
         if include_z
-            kwargs = (;
-                kwargs...,
-                center_z = Fields.field2array(ᶜz),
-                face_z = Fields.field2array(ᶠz),
-            )
+            if ᶜspace.grid.global_geometry isa
+               Geometry.AbstractSphericalGlobalGeometry
+                kwargs = (;
+                    kwargs...,
+                    center_z = Fields.field2array(ᶜz),
+                    face_z = Fields.field2array(ᶠz),
+                    planet_radius = planet_radius,
+                )
+            else
+                kwargs = (;
+                    kwargs...,
+                    center_z = Fields.field2array(ᶜz),
+                    face_z = Fields.field2array(ᶠz),
+                )
+            end
```

Comment:
```suggestion
            kwargs = (;
                kwargs...,
                center_z = Fields.field2array(ᶜz),
                face_z = Fields.field2array(ᶠz),
            )
            if ᶜspace.grid.global_geometry isa
               Geometry.AbstractSphericalGlobalGeometry
                planet_radius = ᶜspace.grid.global_geometry.radius
                kwargs = (; kwargs..., planet_radius)
            end
```
This could be a little less repetitive. Same for the `GrayRadiation` method above.